docs/reference/ccr/getting-started.asciidoc

```diff
@@ -73,22 +73,16 @@ can <<modules-cross-cluster-search,search across clusters>> and set up {ccr}.
 To complete this tutorial, you need:
 
 * A license on both clusters that includes {ccr}. {kibana-ref}/managing-licenses.html[Activate a free 30-day trial].
-* The `read_ccr` cluster privilege and `monitor` and `read` privileges
-for the leader index on the remote cluster. <<stack-management-ccr-remote,Configure remote cluster privileges>>.
-* The `manage_ccr` cluster privilege and `monitor`, `read`, `write` and
-`manage_follow_index` privileges to configure remote clusters and follower
-indices on the local cluster. <<stack-management-ccr-local,Configure local cluster privileges>>.
 * An index on the remote cluster that contains the data you want to replicate.
 This tutorial uses the sample eCommerce orders data set.
 {kibana-ref}/get-started.html#gs-get-data-into-kibana[Load sample data].
 * In the local cluster, all nodes with the `master` <<node-roles,node role>> must
-  also have the <<remote-node,`remote_cluster_client`>> role. The local cluster
-  must also have at least one node with both a data role and the
-  <<remote-node,`remote_cluster_client`>> role. Individual tasks for coordinating
-  replication scale based on the number of data nodes with the
-  `remote_cluster_client` role in the local cluster.
+also have the <<remote-node,`remote_cluster_client`>> role. The local cluster
+must also have at least one node with both a data role and the
+<<remote-node,`remote_cluster_client`>> role. Individual tasks for coordinating
+replication scale based on the number of data nodes with the
+`remote_cluster_client` role in the local cluster.
 
-[[ccr-getting-started-remote-cluster]]
 ==== Connect to a remote cluster
 To replicate an index on a remote cluster (Cluster A) to a local cluster (Cluster B), you configure Cluster A as a remote on Cluster B.
```

Comment:
@ywangd, are you saying that users need the `manage` cluster privilege on the local cluster to connect a remote cluster?

Reply:
For adding or removing a remote cluster, Yes because this is done via the [Update Cluster Settings API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-update-settings.html).

I think there are a few layers for users with different privileges:
1. The very first thing is to establish trust between two clusters. This requires adding CA or cert of the other cluster to this cluster. That means modifying `elasticsearch.yml` and is at the operation system level. A system admin or infrastructure owner needs to do this. Or the two cluster can share a common CA. Nevertheless, this step is before we can even run Elasticsearch.
2. Actually add the remote cluster. This is done via the Settings API which requires `manage` privilege.
3. Creating the users and roles for CCR or CCS. This requires `manage_security`.
4. Actual usage of CCR and CCS. This uses whatever is configured in the last step.

Reply:
I appreciate the specifics you've provided, and am calling out privileges in each of these sections so that users are aware of which privileges are required before even attempting each of those procedures.